### DISCUSSION TITLE
Who was the famous literary wife of archaeologist Sir Max Mallowan?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#393] [Galaxy Trivia] [answered]
**Hint:** The Mousetrap author

**Comment by thuongtruong1009** [comment, marked as answer]
The famous literary wife of archaeologist Sir Max Mallowan was Agatha Christie, one of the most widely read authors of detective fiction in the world. Agatha Christie and Max Mallowan were married in 1930 and remained together until Christie's death in 1976. During their marriage, they worked together on many archaeological expeditions, and Christie's experiences in the Middle East often found their way into her novels. Christie dedicated several of her books to Mallowan and even wrote a memoir, "Come, Tell Me How You Live," about their experiences on the archaeological digs.

**Comment by peterbaccaro** [comment]
The literary wife of archaeologist Sir Max Mallowan was the novelist and playwright **Agatha Christie**.
